docker-compose.yml

```diff
@@ -6,10 +6,16 @@ services:
     ports:
       - '1984:1984'
     healthcheck:
-      test: ['CMD', 'curl', '-f', 'http://arlocal:1984/']
+      test: ['CMD', 'curl', '-f', 'http://localhost:1984/']
```

Comment:
healthcheck is against the local container and doesn't need to go over the docker compose default network